### PR TITLE
created Radio group component

### PR DESCRIPTION
**Pull request by harivikneshs**
Lets user to wrap up multiple radio buttons without worrying about onClick listeners and getting selected item value.

### CODE REVIEW

**Comment by LJcz5qROuAIRsnsOgVrAMwPPMH6vH2Qw2PPEVLm** [comment]
@harivikneshs can you add some screenshots how the component is looking!!

**Review by vinitraj10 — CHANGES_REQUESTED** [review]
@harivikneshs  Please checkout other components docs and write doc for this too,let me know for any help

File: example/src/examples/RadioButtonExample.tsx [hunk 1]
```diff
@@ -22,6 +26,15 @@ const RadioButtonExample = () => {
         justifyContent: 'center',
       }}
     >
+      <Icon
+        name="arrow-back"
+        size={20}
+        style={{ margin: 20 }}
+        onPress={() => props.navigation.pop()}
```
Comment (LJcz5qROuAIRsnsOgVrAMwPPMH6vH2Qw2PPEVLm) [inline example/src/examples/RadioButtonExample.tsx]: pass only referrence

**Review by vinitraj10 — APPROVED** [review]

File: example/src/examples/RadioButtonExample.tsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+import Icon from 'react-native-vector-icons/MaterialIcons';
 
 // eslint-disable-next-line import/no-unresolved
-import { RadioButton, useColors } from 'frontatish';
+import { RadioButton, useColors, RadioGroup } from 'frontatish';
 import { SafeAreaView } from 'react-native-safe-area-context';
 // import { Fonts, getColors } from '../styles';
 
-const RadioButtonExample = () => {
+const RadioButtonExample = (props: any) => {
```
Comment (jDNmVjbwQVhqWFSKHBflhkVN7xo3kpxux4eN0Ki) [inline example/src/examples/RadioButtonExample.tsx]: define type of prop

**Review by amanBhawsar — CHANGES_REQUESTED** [review]

**Review by vinitraj10 — APPROVED** [review]

**Pull request merged**